File: Majo/Act-Instructor_Henry/lenguajes/ruby/mayo/29-05-24.rb
puts "Ingresa un número entero"
b = gets.chomp.to_i

if b >0
    puts "El número #{b} es positivo"
else
    if b <0
        puts "El número #{b} es negativo"
    else 
        puts "El número es cero"    
    end 
end                

File: Majo/Act-Instructor_Henry/lenguajes/dart/mayo/29-05-24.dart
import 'dart:io';

void main() {
  print('Ingresa un número entero');
  int b = int.parse(stdin.readLineSync()!);

  if (b >0) {
    print('El número $b es positivo');
  } else {
    if (b <0){
      print('El número es $b es negativo');
    } else {
        print('El número es cero');
    }
  }
}